chips/sam4l/src/adc.rs

```diff
@@ -408,12 +447,19 @@ impl hil::adc::Adc for Adc {
             self.active.set(true);
             self.continuous.set(true);
 
+            let trgsel;
+            if self.cpu_clock.get() {
+                trgsel = 1; // internal timer trigger
+            } else {
+                trgsel = 3; // continuous mode
```

Comment:
This was addressed in a different comment. It was accounted for in setting the maximum frequency for RCSYS and RC32K to 1/32nd of their expected frequency. Factor of 8 for conversion and factor of 4 because it is the minimum PRESCAL valus (DIV4).